content/en/docs/languages/java/automatic/configuration.md

```diff
@@ -4,7 +4,7 @@ linkTitle: Configuration
 weight: 10
 aliases: [agent-config]
 # prettier-ignore
-cSpell:ignore: akka armeria classloaders couchbase Customizer datasource dbcp Dotel dropwizard dubbo enduser finatra hikari hikaricp HSET httpasyncclient httpclient hystrix jaxrs jaxws jedis jodd kotlinx logback logmanager mojarra myfaces okhttp oshi pekko rabbitmq ratpack rediscala redisson restlet rocketmq serverlessapis spymemcached twilio vaadin vertx vibur webflux webmvc
+cSpell:ignore: akka armeria authservice classloaders couchbase Customizer datasource dbcp Dotel dropwizard dubbo enduser finatra hikari hikaricp HSET httpasyncclient httpclient hystrix jaxrs jaxws jedis jodd kotlinx logback logmanager LOWMEMORY mojarra myfaces myservice okhttp oshi ottrace pekko PKCS rabbitmq ratpack rediscala redisson restlet retryable rocketmq serverlessapis spymemcached twilio vaadin vertx vibur webflux webmvc
 ---
 
 ## SDK Autoconfiguration
```

Comment:
Its a bit strange that the SDK autoconfiguration section is nested under "Language APIs & SDKs -> Java -> Automatic", where automatic is a page focussed on the otel java agent. SDK autoconfiguration is used within the otel java agent, but can also be used independently, and we actively encourage that.

Is it crazy to have a dedicated page on SDK configuration?

Reply:
And now there is no longer an `Automatic` section (since  https://github.com/open-telemetry/opentelemetry.io/pull/4430) and instead this now lives in a new location, `Zero-code Instrumentation -> Java -> Agent -> Configuration`

As a user trying to configure the SDK, i'm not sure it would be obvious to me to look there to try and find the configuration options.

Looking at some other languages, I don't see too much of a common pattern for where the language specific SDK configuration lives.

* `.NET` uses the structure that java used to adhere to 
	* `.NET -> Automatic -> Configuration` ([link](https://opentelemetry.io/docs/languages/net/automatic/configuration/))
* `Python` uses the the structure that java used to adhere to 
	* `Python -> Automatic -> Configuration` ([link](https://opentelemetry.io/docs/languages/python/automatic/configuration/))
* `Swift` has an "SDK Configuration" section on their instrumentation page
	* `Swift -> Instrumentation` ([link](https://opentelemetry.io/docs/languages/swift/instrumentation/#sdk-configuration))
* `PHP` has a page for the SDK with the configs
	* `PHP -> SDK` ([link](https://opentelemetry.io/docs/languages/php/sdk/#configuration))
* `C++` doesn't have their SDK documentation within the doc site, but instead [they link out](https://opentelemetry.io/docs/languages/cpp/instrumentation/#further-reading-1)
* `Erlang`, `Rust`, `Go`, `Javascript` and `Ruby`  don't have any specific page focused on SDK configuration options documented within the doc site that I could find

To me it would make sense to have this live under something like `Java -> SDK` similar to PHP, or `Java -> Configuration`

Reply:
Hey, commenting on that to clarify some confusion: we are currently in the process of removing the navigation of  `<Language> -> Automatic` for **all** languages, we just started with Java: the term "automatic" and "automatic instrumentation" is highly misleading (as this example shows) and we want to reduce that confusion.

So the ".NET" an "Python" thing is going away.

I agree with @jack-berg that this should be a subpage of "Language APIs & SDKs > Java > " directly and not part of zero code instrumentation.

Reply:
Thanks for the clarification @svrnm, are there any defined patterns I should follow around the new subpage and things like the page name (`SDK` vs `Configuration` vs `SDK Configuration`) to help with consistency?

Reply:
I would suggest `Configuration` since this is a subsection of `Language APIs & **SDKs** > **Java** >`